tests/test_acceptparse.py

```diff
@@ -466,1570 +475,426 @@ def test_parse_offer__invalid(self, offer):
-        ],
-    )
-    def test_acceptable_offers__valid_offers(
-        self, header_value, offers, expected_returned
-    ):
-        instance = AcceptValidHeader(header_value=header_value)
-        returned = instance.acceptable_offers(offers=offers)
-        assert returned == expected_returned
-
-    def test_acceptable_offers_uses_AcceptOffer_objects(self):
-        from webob.acceptparse import AcceptOffer
-
-        offer = AcceptOffer("text", "html", (("level", "1"),))
-        instance = AcceptValidHeader(header_value="text/*;q=0.5")
-        result = instance.acceptable_offers([offer])
-        assert result == [(offer, 0.5)]
-
-    @pytest.mark.filterwarnings(IGNORE_BEST_MATCH)
-    def test_best_match(self):
-        accept = AcceptValidHeader("text/html, foo/bar")
-        assert accept.best_match(["text/html", "foo/bar"]) == "text/html"
-        assert accept.best_match(["foo/bar", "text/html"]) == "foo/bar"
-        assert accept.best_match([("foo/bar", 0.5), "text/html"]) == "text/html"
-        assert accept.best_match([("foo/bar", 0.5), ("text/html", 0.4)]) == "foo/bar"
-
-    @pytest.mark.filterwarnings(IGNORE_BEST_MATCH)
-    def test_best_match_with_one_lower_q(self):
-        accept = AcceptValidHeader("text/html, foo/bar;q=0.5")
-        assert accept.best_match(["text/html", "foo/bar"]) == "text/html"
-        accept = AcceptValidHeader("text/html;q=0.5, foo/bar")
-        assert accept.best_match(["text/html", "foo/bar"]) == "foo/bar"
-
-    @pytest.mark.filterwarnings(IGNORE_BEST_MATCH)
-    def test_best_match_with_complex_q(self):
-        accept = AcceptValidHeader("text/html, foo/bar;q=0.55, baz/gort;q=0.59")
-        assert accept.best_match(["text/html", "foo/bar"]) == "text/html"
-        accept = AcceptValidHeader("text/html;q=0.5, foo/bar;q=0.586, baz/gort;q=0.596")
-        assert accept.best_match(["text/html", "baz/gort"]) == "baz/gort"
-
-    @pytest.mark.filterwarnings(IGNORE_BEST_MATCH)
-    def test_best_match_json(self):
-        accept = AcceptValidHeader("text/html, */*; q=0.2")
-        assert accept.best_match(["application/json"]) == "application/json"
-
-    @pytest.mark.filterwarnings(IGNORE_BEST_MATCH)
-    def test_best_match_mixedcase(self):
-        accept = AcceptValidHeader(
-            "image/jpg; q=0.2, Image/pNg; Q=0.4, image/*; q=0.05"
-        )
-        assert accept.best_match(["Image/JpG"]) == "Image/JpG"
-        assert accept.best_match(["image/Tiff"]) == "image/Tiff"
-        assert (
-            accept.best_match(["image/Tiff", "image/PnG", "image/jpg"]) == "image/PnG"
-        )
-
-    @pytest.mark.filterwarnings(IGNORE_BEST_MATCH)
-    @pytest.mark.filterwarnings(IGNORE_CONTAINS)
-    def test_best_match_zero_quality(self):
-        assert (
-            AcceptValidHeader("text/plain, */*;q=0").best_match(["text/html"]) is None
-        )
-        assert "audio/basic" not in AcceptValidHeader("*/*;q=0")
```

Comment:
These asserts seem to be worth keeping - I don't think they or any functionally equivalent asserts are in the new tests? (One assert is for `.best_match` and the other is for `.__contains__` - not sure why the `.__contains__` one was placed into a `.best_match` test method.)